src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -0,0 +1,194 @@
+<link rel="import" href="../kano-style/color.html">
+<dom-module id="kw-social-comment">
+    <template>
+        <style>
+             :host {
+                display: block;
+                @apply --layout-vertical;
+                @apply --layout-center;
+            }
+            
+            .orange-button {
+                background-color: var(--color-orange);
+                border-radius: 5px;
+                width: 79px;
+                height: 32px;
+                color: var(--color-white);
+                border: none;
+            }
+            
+            .comment-button {
+                width: 100px;
+            }
+            
+            .comment-button img {
+                position: relative;
+                top: 2px;
+                left: 2px;
+            }
+            
+            .comments-box {
+                @apply --layout-vertical;
+                @apply --layout-center;
+                @apply --layout-justified;
+                background-color: var(--color-grey-lightest);
+                width: 100%;
+                padding-bottom: 54px;
+            }
+            
+            .comments-box h3 {
+                color: var(--color-grey-dark);
+                margin: 50px;
+                font-size: 16px;
+                line-height: 43px;
+            }
+            
+            .comments-box .inputcomment {
+                @apply --layout-horizontal;
+                @apply(--layout-center);
+                width: 80%;
+                margin: 32px;
+            }
+            
+            .comment-avatar {
+                @apply --layout-flex-1;
+                margin-left: 16px;
+                margin-right: 16px;
+            }
+            
+            .comments-box .inputcomment .commentbox {
+                @apply --layout-horizontal;
+                @apply(--layout-center);
+                border-radius: 3px;
+                border: 1px solid var(--color-grey-lighter);
+                height: 45px;
+                width: 90%;
+                padding: 8px;
+                background-color: var(--color-grey-lightest);
+            }
+            
+            .comments-box .inputcomment .commentbox input {
+                width: 80%;
+                height: 32px;
+                font-size: 14px;
+                border: none;
+                margin: 8px;
+            }
+            
+            .comments-box .line {
+                height: 1px;
+                width: 90%;
+                display: block;
+                background-color: var(--color-white);
+                opacity: 0.2;
+            }
+            
+            .comment {
+                width: 80%;
+                height: 100px;
+                @apply --layout-horizontal;
+                @apply(--layout-center);
+            }
+            
+            .content {
+                font-size: 16px;
+                font-family: "Bariol";
```

Comment:
can you import the kano style typography and use font-family: var(--font-body); please?